dspace-api/pom.xml

```diff
@@ -864,5 +877,49 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <!-- JClouds Assetstorage Support -->
+        <dependency>
+            <groupId>org.apache.jclouds</groupId>
+            <artifactId>jclouds-core</artifactId>
+            <version>2.6.0</version>
```

Comment:
If all these JClouds dependencies should be "synced" in terms of the version, then we may want to pull this out into a property named `jclouds.version` along all the other `version` properties in our main POM: https://github.com/DSpace/DSpace/blob/main/pom.xml#L31-L59.

This is optional, but I wanted to call it out as it might make future updates to this dependency a bit easier.